tests/cross_functional/ui/fusion_access/test_virtualmachine_lifecycle.py

```
@@ -0,0 +1,247 @@
+        logger.info("-" * 80)
+        self.base_ui.take_screenshot("customization_page")
+
+        self.vm_ui.click_customization_storage_tab()
+        self.base_ui.take_screenshot("customization_storage_tab")
+        time.sleep(2)
+
+        self.vm_ui.click_rootdisk_kebab_and_edit()
+        self.base_ui.take_screenshot("edit_disk_popup_opened")
+
+        storage_class = self.vm_ui.change_storageclass_to_vm_option()
+        logger.info(f"Changed StorageClass to: {storage_class}")
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)